docs/content/development/extensions-core/kafka-ingestion.md

```diff
@@ -206,13 +206,27 @@ Suspend indexing tasks associated with a supervisor. Note that the supervisor it
 operating and emitting logs and metrics, it will just ensure that no indexing tasks are running until the supervisor
 is resumed. Responds with updated SupervisorSpec.
 
-#### Resume Supervisor 
+#### Suspend All Supervisors
+
+```
+POST /druid/indexer/v1/supervisor/suspendAll
```

Comment:
It's fine for me if change to `/druid/indexer/v1/supervisors/suspend` , but just as you said before, that seems more painful than useful to change. So I keep this `All` suffix. Besides, in PR #6185 `All` is also be used in endpoint path.

Reply:
Ah, I wasn't suggesting _supervisors_ with the _s_ since yeah that's a more dramatic change, just leaving out the _All_, but looking around at the rest of the APIs some more, and things already aren't terribly consistent with things like pluralization, etc., anyway so I think basically whatever is probably fine as long as there are docs and stuff does what is intended 😜